exercises/binary_converter.cpp

```diff
@@ -5,3 +5,25 @@
   Insert first number: 8
   The binary number is: 1000
 */
+
+#include <iostream>
+using namespace std;
+
+string convertToBinary(int x){ 
+  while (x > 0){
+    string num = to_string(x % 2);
+    x /= 2; 
+    return convertToBinary(x) + num;
+  }
+  return "\n";
+}
+
+int main(){
+  int x = 16;
+
+  string binary = convertToBinary(x);
+
+  cout << "Il numero: " << x << " in binario è " << binary << endl;
+  
+  return 0;
```

Comment:
Potresti incrementare la leggibilità del codice con la seguente modifica:

```cpp
#define EXIT_SUCCESS 0;
```

```suggestion
  return EXIT_SUCCESS;
```

Oppure quelle definite già nell'apposita [libreria](https://en.cppreference.com/w/cpp/utility/program/EXIT_status.html)